deno_dist/middleware/bearer-auth/index.ts

```diff
@@ -23,6 +23,35 @@ type BearerAuthOptions =
       hashFunction?: Function
     }
 
+/**
+ * Bearer authentication middleware for Hono.
+ *
+ * @see {@link https://hono.dev/middleware/builtin/bearer-auth}
+ *
+ * @param {BearerAuthOptions} options - The options for the bearer authentication middleware.
+ * @param {string | string[]} [options.token] - The string or array of strings to validate the incoming bearer token against.
+ * @param {Function} [options.verifyToken] - The function to verify the token.
+ * @param {string} [options.realm=""] - The domain name of the realm, as part of the returned WWW-Authenticate challenge header.
+ * @param {string} [options.prefix="Bearer"] - The prefix (or known as `schema`) for the Authorization header value.
+ * @param {string} [options.headerName=Authorization] - The header name.
+ * @param {Function} [options.hashFunction] - A function to handle hashing for safe comparison of authentication tokens.
+ * @returns {MiddlewareHandler} - The middleware handler function.
```

Comment:
The hyphen is not necessary.

![CleanShot 2024-05-17 at 06 58 55@2x](https://github.com/honojs/hono/assets/10682/88cc0100-81e0-4d09-88b8-7112b8342fbe)


So it should be:

`* @returns {MiddlewareHandler} The middleware handler function.`

Reply:
Resolved and squashed to first commit